test/e2e/suite/conformance/certificates/acme/acme.go

```diff
@@ -28,76 +28,72 @@ import (
 	"github.com/jetstack/cert-manager/test/e2e/framework/addon/pebble"
 	"github.com/jetstack/cert-manager/test/e2e/framework/addon/tiller"
 	"github.com/jetstack/cert-manager/test/e2e/suite/conformance/certificates"
+	"github.com/jetstack/cert-manager/test/e2e/suite/issuers/acme/dnsproviders"
 )
 
 var _ = framework.ConformanceDescribe("Certificates", func() {
 	// unsupportedFeatures is a list of features that are not supported by the ACME
 	// issuer type using HTTP01
 	var unsupportedFeatures = certificates.NewFeatureSet(
 		certificates.IPAddressFeature,
-		certificates.Wildcards,
+		certificates.WildcardsFeature,
+		certificates.DurationFeature,
 	)
 
-	provisioner := &acmeIssuerProvisioner{setGroupName: false}
+	provisionerHTTP01 := new(acmeIssuerProvisioner)
 	(&certificates.Suite{
 		Name:                "ACME HTTP01",
-		CreateIssuerFunc:    provisioner.create,
-		DeleteIssuerFunc:    provisioner.delete,
+		CreateIssuerFunc:    provisionerHTTP01.createHTTP01,
+		DeleteIssuerFunc:    provisionerHTTP01.delete,
 		UnsupportedFeatures: unsupportedFeatures,
 	}).Define()
 
-	// crProvisioner sets the issuerRef.group field on Certificates it creates
-	crProvisioner := &acmeIssuerProvisioner{setGroupName: true}
+	provisionerDNS01 := new(acmeIssuerProvisioner)
 	(&certificates.Suite{
-		Name:                "ACME HTTP01 (CertificateRequest)",
-		CreateIssuerFunc:    crProvisioner.create,
-		DeleteIssuerFunc:    crProvisioner.delete,
+		Name:                "ACME DNS01",
+		CreateIssuerFunc:    provisionerDNS01.createDNS01,
+		DeleteIssuerFunc:    provisionerDNS01.delete,
 		UnsupportedFeatures: unsupportedFeatures,
 	}).Define()
 })
 
 type acmeIssuerProvisioner struct {
```

Comment:
It may be better to create a separate `dns01` version of this struct? Not too sure, and not *too* fussed either way